deep_qa/models/sequence_tagging/verb_semantics_model.py

```diff
@@ -0,0 +1,65 @@
+from keras.layers import Dense, Input, TimeDistributed, Multiply, Add
+
+from deep_qa.layers.encoders.bag_of_words import BOWEncoder
+from overrides import overrides
+
+from deep_qa.data.instances.sequence_tagging.verb_semantics_instance import VerbSemanticsInstance
+from deep_qa.common.params import Params
+from deep_qa.data.instances.sequence_tagging import concrete_instances
+from deep_qa.training.text_trainer import TextTrainer
+from deep_qa.training.models import DeepQaModel
+
+class VerbSemanticsModel(TextTrainer):
+    """
+    This ``VerbSemanticsModel`` encodes a sequence of text with some number of stacked
+    ``seq2seq_encoders``, then predicts a tag at each index.
+
+    Parameters
+    ----------
+    num_stacked_rnns : int, optional (default: ``1``)
+        The number of ``seq2seq_encoders`` that we should stack on top of each other before
+        predicting tags.
+    instance_type : str
+        Specifies the particular subclass of ``TaggedSequenceInstance`` to use for loading data,
+        which in turn defines things like how the input data is formatted and tokenized.
+    """
+    def __init__(self, params: Params):
+        self.num_stacked_rnns = params.pop('num_stacked_rnns', 1)
+        instance_type = params.pop('instance_type', "VerbSemanticsInstance")
+        self.instance_type = concrete_instances[instance_type]
+        super(VerbSemanticsModel, self).__init__(params)
```

Comment:
@bhavanadalvi: re matt's comment in his email, you need to add the following line here:

`params.setdefault('validation_metric', 'val_loss')`

This is because we are using some Keras functionality which allows us to save a model when some metric has fallen/improved. There is a default for this in keras if your model has a single output, but as yours has 2, you need to set it explicitly, before you call the super class.